docs/GettingStarted.md

```diff
@@ -324,7 +325,8 @@ For a list of available integrations, and their configuration options, please re
 
 | Name                     | Key                        | Versions Supported       | How to configure                    | Gem source                                                                     |
 | ------------------------ | -------------------------- | ------------------------ | ----------------------------------- | ------------------------------------------------------------------------------ |
-| Action View              | `action_view`              | `>= 3.2`          | *[Link](#action-view)*              | *[Link](https://github.com/rails/rails/tree/master/actionview)*                |
+| Action Cable             | `action_cable`             | `>= 5.0, < 6.0`          | *[Link](#action-cable)*             | *[Link](https://github.com/rails/rails/tree/master/actioncable)*               |
```

Comment:
Are these version support numbers right? I thought this was supported in Rails 6.

Reply:
Oops, fixed this.